php/libraries/User.class.inc

```diff
@@ -482,37 +482,29 @@ class User extends UserPermissions
      * database.
      *
      * @param \Password $password The plain text password to be hashed and saved.
-     * @param ?DateTime $expiry   When the password will expire. Can be set to a
-     *                            date in the past in order to create a one-time
-     *                            password.
+     * @param bool      $expired  Whether the password is expired. If so, a reset
+     *                            will be triggered on the user's next login.
      *
      * @return void
      */
     function updatePassword(
         \Password $password,
-        ?\DateTime $expiry = null
+        bool $expired = false
```

Comment:
does this need to have ?bool specified if the $expired argument isn't provided in the function call?

Reply:
I don't think so. The `?` signals that a variable can be null which isn't the case here. If an argument isn't provided, it'll default to `false`.